src/openapi/I_Test_Driver_FdV.yaml

```diff
@@ -2226,6 +2273,7 @@ components:
         - emergency
         - eab
         - dental
+        - childsrecord        
```

Comment:
**question**: Zum Verständnis. Was ist der Unterschied zwischen `childsrecord` und `child`?

Reply:
Nach einer Aktenkontomigration von ePA 2.6 zu ePA 3.0 können im ePA 3.0 Aktenkonto Dokumente der Kategorie childsrecord liegen. 
Die Kategorie wurde in Release 3.0.2-1 neu in die Legal Policy aufgenommen. (siehe gemSpec_Aktensystem_ePAfueralle_V1.2.0  
A_24562 - XDS Document Service - Migration: Auflösung abgekündigter Ordner
Der XDS Document Service MUSS im Zuge der Migration von ePA 2.6 auf ePA 3.0 die 
abgekündigten Kategorien auflösen. Dabei MÜSSEN sämtliche Dokumente gemäß der 
Einordnungsregeln in A_19388-* neu Ordnern zugeordnet werden und die Ordner der 
abgekündigten Kategorien gelöscht werden.[<=]
Die in ePA 2 angelegten dynamischen Ordner der Kategorie childsrecord können 
Kinder identifizieren, deren Daten nicht in ihren eigenen Akten gehalten wurden. Diese 
dynamischen Ordner sind nach folgender Regel in ePA 2 vom Primärsystem angelegt 
worden: Folder.title wurde mit dem Namen und Geburtsdatum des Kindes belegt. 
Bildungsregel: Nachname + ", " + 1. Vorname + " "Datum im Format TT.MM.YYYY. 
Beispiel: "Musterkind, Max 03.03.2017".
Die Kinderuntersuchungshefte werden nicht migriert und verbleiben im Ordner 
childsrecord.)